cni/network/network.go

```diff
@@ -1258,18 +1259,25 @@ func convertNnsToCniResult(
 
 			intIndex := i
 			for _, ip := range ni.Ipaddresses {
+				ipAddr := net.ParseIP(ip.Ip)
 
-				ipWithPrefix := fmt.Sprintf("%s/%s", ip.Ip, ip.PrefixLength)
-				_, ipNet, err := net.ParseCIDR(ipWithPrefix)
+				prefixLength, err := strconv.Atoi(ip.PrefixLength)
 				if err != nil {
-					log.Printf("Error while converting to cni result for %s operation on pod %s. %s",
-						operationName, podName, err)
+					log.Printf("Error parsing prefixLength (%s) for %s operation on pod %s, err:%s.",
```

Comment:
we moved to zap logging meanwhile.. you can pull latest master and update it

Reply:
Pulled latest changes and updated to zap logging.